var/spack/repos/builtin/packages/python/package.py

```diff
@@ -104,96 +104,6 @@ class Python(Package):
     version("3.8.2", sha256="e634a7a74776c2b89516b2e013dda1728c89c8149b9863b8cea21946daf9d561")
     version("3.8.1", sha256="c7cfa39a43b994621b245e029769e9126caa2a93571cee2e743b213cceac35fb")
     version("3.8.0", sha256="f1069ad3cae8e7ec467aa98a6565a62a48ef196cb8f1455a245a08db5e1792df")
-    version(
-        "3.7.17",
-        sha256="fd50161bc2a04f4c22a0971ff0f3856d98b4bf294f89740a9f06b520aae63b49",
-        deprecated=True,
-    )
```

Comment:
Literally nothing is gained by removing this, no simplifications of the build or whatever. This change solely complicates bootstrapping

Reply:
I kind of agree with the statement above. When preparing a new set of binaries for bootstrapping in https://github.com/spack/spack-bootstrap-mirrors, testing on a pristine Python 3.6 was more difficult than needed because the Spack under test couldn't install Python 3.6.

> To go a step farther, I don't think Spack should support running on Python 3.6 or 3.7

As you say this is another, related, discussion. For what is worth, if we decided that we don't support running on Python 3.7 all this would be fine. But I think removing this support and deleting the latest Python 3.7 version should go together.